app/s3_client/s3_mou_client.py

```diff
@@ -4,17 +4,16 @@
 from app.s3_client.s3_logo_client import get_s3_object
 
 
-def get_mou(organisation_is_crown):
+def get_mou():
     bucket = current_app.config['MOU_BUCKET_NAME']
-    filename = 'crown.pdf' if organisation_is_crown else 'non-crown.pdf'
-    attachment_filename = 'U.S. Notify data sharing and financial agreement{}.pdf'.format(
-        '' if organisation_is_crown else ' (non-crown)'
+    filename = 'agreement.pdf'
+    attachment_filename = 'U.S. Notify data sharing and financial agreement.pdf'.format(
```

Comment:
[suggestion] looks like there's nothing in the format anymore, so lets get rid of it in favor of a string literal